lib/keystore.js

```diff
@@ -57,17 +60,22 @@ function nacl_decodeHex(msgHex) {
 }
 
 
-var KeyStore = function(mnemonic, pwDerivedKey, hdPathString) {
-
-  this.defaultHdPathString = "m/0'/0'/0'";
+var KeyStore = function(mnemonic, pwDerivedKey, hdPathString, salt) {
```

Comment:
Yes, I’ve now added the console.warn call!

> On Sep 5, 2016, at 1:36 PM, Christian Lundkvist <notifications@github.com> wrote:
> 
> In lib/keystore.js <https://github.com/ConsenSys/eth-lightwallet/pull/113#discussion_r77554725>:
> 
> > @@ -57,17 +60,22 @@ function nacl_decodeHex(msgHex) {
> >  }
> >  
> >  
> > -var KeyStore = function(mnemonic, pwDerivedKey, hdPathString) {
> > -
> > -  this.defaultHdPathString = "m/0'/0'/0'";
> > +var KeyStore = function(mnemonic, pwDerivedKey, hdPathString, salt) {
> Yes, exactly: console.warn() would still make it backward compatible but guide the user to upgrade!
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/ConsenSys/eth-lightwallet/pull/113/files/0c8f7be3281f39762aaee2bcc4c44e59bfc62be1#r77554725>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAhIj6Yi-p6qRV4irVMplJbhLyKNDq48ks5qnH1ngaJpZM4JyK7d>.
>